ENH Adds nan passthrough in OrdinalEncoder (#19069)
scikit-learn/scikit-learn: doc/modules/preprocessing.rst

the categories as being ordered, which is often not desired (i.e. the set of
browsers was ordered arbitrarily).

> :class:`OrdinalEncoder` will also passthrough missing values::

Reviewer: Please also document None

the categories as being ordered, which is often not desired (i.e. the set of
browsers was ordered arbitrarily).

> :class:`OrdinalEncoder` will also passthrough missing values. Note that
> missing values indicated by `np.nan` or `None` will both be mapped to
> `np.nan`::

Reviewer: ```suggestion
:class:`OrdinalEncoder` will also passthrough missing values. Missing values
can be indicated by `np.nan` or `None`, which will both be mapped to
`np.nan`::
```
